bot/message_handler/bot.go

```diff
@@ -13,7 +13,6 @@ import (
 
 type IMessageHandler interface {
 	HandleMessage(ctx context.Context, message string) (*MessageDTO, error)
-
```

Comment:
Reverse unecessary changes please, keep in mind the lint check

Reply:
Agree with you, bro